### DISCUSSION TITLE
galaxy badge 2

### DISCUSSION THREAD

**Discussion by xommmax** [Schweinepriester/achievement-testarea#28] [Q&A] [answered]
and again

**Comment by severdia** [comment, marked as answer]
That's it!

**Reply by xommmax** [reply]
yeah, lets check out

**Reply by xommmax** [reply]
@severdia yeees, congrats!
now help me out 😂🥰

**Reply by xommmax** [reply]
u can create dis here https://github.com/Schweinepriester/achievement-testarea/discussions

**Comment by pulkitxm** [comment]
best Answer
